hooks/price/price.go

```diff
@@ -17,6 +17,11 @@ import (
 	"github.com/bandprotocol/chain/v2/x/oracle/types"
 )
 
+const (
+	ASK_COUNT = 16
+	MIN_COUNT = 10
+)
+
```

Comment:
I think we can move this to Hook struct, and let's NewHook receive these value to create from bandd start flag.